exec/cfg.c

```diff
@@ -996,13 +1036,16 @@ static void message_handler_req_lib_cfg_killnode (
 
 	(void)api->totem_mcast (&iovec, 1, TOTEM_SAFE);
 
+send_response:
 	res_lib_cfg_killnode.header.size = sizeof(struct res_lib_cfg_killnode);
 	res_lib_cfg_killnode.header.id = MESSAGE_RES_CFG_KILLNODE;
-	res_lib_cfg_killnode.header.error = CS_OK;
+	res_lib_cfg_killnode.header.error = error;
 
 	api->ipc_response_send(conn, &res_lib_cfg_killnode,
 				    sizeof(res_lib_cfg_killnode));
 
+	free(status);
```

Comment:
This can lead to freeing unitialized `status`. setting `status` to NULL during declaring variable should be enough. Also please consider renaming `status` to something like `status_str` to make more obvious that it is not some kind of error/integer variable.

Reply:
@jfriesse Thanks for pointing this
Changed!